src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -488,6 +488,13 @@ private OidcConfiguration buildOidcConfiguration() {
         // conf.setPreferredJwsAlgorithm(JWSAlgorithm.HS256);
         // set many more as needed...
 
+        // TODO if we set PreferredJwsAlgo then check for the validation
+        // if(conf.getPreferredJwsAlgorithm() != null && OicAlgorithmValidator.isAlgorithmNotFipsCompliant(
+        //                conf.getPreferredJwsAlgorithm().getName())){
+        //            throw new
+        // IllegalArgumentException(Messages.OicConfigNonCompliantAlgo_ErrorMessage(conf.getPreferredJwsAlgorithm().getName()));
+        //        }
+
```

Comment:
```suggestion
```

Reply:
Removed this uncommented code, as no algorithm is used here.